README.md

```diff
@@ -6,30 +6,22 @@ This repository contained code for training and using a model for detecting rein
 
 In this repository we are using [detectron2](https://github.com/facebookresearch/detectron2). To install it follow the procedure:
 
-- Create a virtual environment:
+- If `poetry` is not install, install it by running:
 
 ```bash
-python3 -m venv .venv
-source .venv/bin/activate
+pipx install poetry
 ```
 
-- Install detectron2 and the other dependancies:
+- Make the Python scripts executable by changing their file permissions
 
+```bash
+chmod +x src/dataset.py src/evaluate.py src/predict.py src/train.py src/app.py 
 ```
```

Comment:
You can drop this section; make your file executable on your PC, then add them and make a commit if there is any change. git keeps tracks of these permissions.